plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/functions.tsx

```diff
@@ -43,9 +46,37 @@ function getStatusBadges( subscription: Subscription ): StatusBadge[] {
 		badges.push( {
 			text: __( 'No subscription', 'woocommerce' ),
 			level: StatusLevel.Error,
-			explanation: __(
-				'To get updates and support for this extension, you need to purchase a new subscription, or else share or transfer a subscription for this extension from another account.',
-				'woocommerce'
+			explanation: createInterpolateElement(
+				__(
+					'To receive updates and support, please <purchase>purchase</purchase> a subscription or use a subscription from another account by <sharing>sharing</sharing> or <transferring>transferring</transferring>.',
+					'woocommerce'
+				),
+				{
+					purchase: (
+						<a
+							href={ subscribeUrl( subscription ) }
+							rel="nofollow noopener noreferrer"
+						>
+							renew
+						</a>
+					),
```

Comment:
Might be nice to have tracking on this and the "renew" link in the other tooltip, but we can come back to it later.

Reply:
Can we do this in another PR? I don't want to add tracking until @raicem merges #41365